### DISCUSSION TITLE
When does the firebase token get updated?

### DISCUSSION THREAD

**Discussion by RaedBaker95** [invertase/react-native-firebase#6587] [Q&A] [answered]
Hello Im trying to create service to store fcm token and link it with a user
should i update the fcm token on regular basis

and when does the firebase token get updated?

**Comment by Yawanseh** [comment, marked as answer]
Hi Raed, 

As per Firebase docs, 
it's important to save the token to the server and update the timestamp whenever it changes, such as when:

- The app is restored on a new device
- The user uninstalls/reinstalls the app
- The user clears app data.

ref: https://firebase.google.com/docs/cloud-messaging/manage-tokens
